lib/remap-lang/src/value.rs

```diff
@@ -15,6 +15,7 @@ use std::str::FromStr;
 
 pub use kind::Kind;
 
+#[allow(clippy::large_enum_variant)]
```

Comment:
It'd be nice to add comments for these allows about why we allow them.

Reply:
I checked `regex` code for understanding why we actually get this error. So `regex` use [thread_local](https://crates.io/crates/thread_local) for some cache. Version `1.1.0` of `thread_local` was released on `Jan 8, 2021` and one of the changes was inline data to structure from the `Box`:
https://github.com/Amanieu/thread_local-rs/blob/v1.0.1/src/lib.rs#L95
vs
https://github.com/Amanieu/thread_local-rs/blob/v1.1.0/src/lib.rs#L107
Changes were made in this PR: https://github.com/Amanieu/thread_local-rs/pull/22
I'm not sure how this change will affect `vector` memory consumption, so will revert `thread_local` update and create a separate issue for this.